presto-main/src/main/java/com/facebook/presto/sql/planner/PlanFragment.java

```diff
@@ -144,6 +147,14 @@ public StatsAndCosts getStatsAndCosts()
         return statsAndCosts;
     }
 
+    @JsonProperty
+    public Optional<String> getJsonRepresentation()
```

Comment:
What's the purpose of this field? It's not clear from this commit. Also, it seems odd for this class, which is meant to be a structural representation of the plan, to contain anything related to a rendering/representation for display purposes.